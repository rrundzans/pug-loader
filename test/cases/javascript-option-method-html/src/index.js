const html = require(`Template/widget.pug?{"b":"yyy"}`);

console.log(html);