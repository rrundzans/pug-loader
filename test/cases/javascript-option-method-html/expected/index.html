<h1>Hello!</h1><span>Param[a]: 7</span><span>Param[b]: yyy</span><img src="assets/images/image.f78b30f4.jpeg">